io_scene_niftools/modules/nif_import/property/object/__init__.py

```diff
@@ -83,9 +82,9 @@ def import_extra_datas(self, root_block, b_obj):
             elif isinstance(n_extra, NifFormat.BSXFlags):
                 b_obj.niftools.bsxflags = n_extra.integer_data
             elif isinstance(n_extra, NifFormat.BSInvMarker):
-                b_obj.niftools_bs_invmarker.add()
-                b_obj.niftools_bs_invmarker[0].name = n_extra.name.decode()
-                b_obj.niftools_bs_invmarker[0].bs_inv_x = (-n_extra.rotation_x / 1000) % (2 * pi)
-                b_obj.niftools_bs_invmarker[0].bs_inv_y = (-n_extra.rotation_y / 1000) % (2 * pi)
-                b_obj.niftools_bs_invmarker[0].bs_inv_z = (-n_extra.rotation_z / 1000) % (2 * pi)
-                b_obj.niftools_bs_invmarker[0].bs_inv_zoom = n_extra.zoom
+                bs_inv_item = niftools_scene.bs_inv.add()
```

Comment:
I still think that we should enforce '1 scene = 1 nif' because
- easier to handle, easier export logic as nif root data always becomes scene data; no need to iterate over various objects to try to find root data
- user can't accidentally set root extra data on the wrong b obj

Reply:
I've moved it back to object